week-4/A-objects-intro/exercise-part-2.js

```diff
@@ -5,17 +5,18 @@ The objects below have some syntax issues - try and fix them all!
 */
 
 var kitten = {
-    fur colour: "orange",
-    age "23"
+    furcolour: "orange",
+    age: 23
 };
 
-var laptop = 
-    brand: "Lenovo"
-    ram "5GB"
+var laptop = {
+    brand: "Lenovo",
+    ram: "5GB"
 }
 
 var phone = {
-    operating system "iOS",
+    operatingsystem: "iOS",
     hasStylus: true,
-    megapixels 12
-    "batteryLife": "24 hours"
\ No newline at end of file
+    megapixels: 12,
+    batteryLife: "24 hours"
+}
```

Comment:
Great.  
There should be a semi-colon at the end of each variable declaration.   `var x = {k: v};`